Aptoide/aptoide-client-v8
# [ASV-1377] MoPub in prod banners

**What does this PR do?**

**Note: Please accept the mopub interstitial ad first before reviewing this PR**

   This PR aims at adding the mopub banners in production. These banners should be located on 3 different places: home, search and appview.

**Database changed?**

   No

**Where should the reviewer start?**

- [ ] Home.java

**How should this be manually tested?**
Open those aforementioned views and check if you see any banner. Notice that you will probably need to use a vpn

**What are the relevant tickets?**

  Tickets related to this pull-request: [ASV-1377](https://aptoide.atlassian.net/browse/ASV-1377)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.) 




**Code Review Checklist**
